### ISSUE TITLE
[BUG] Changing config/php/php-local.ini doesn't have any effect

### ISSUE THREAD

**Issue by thechubbypanda** [linuxserver/docker-nextcloud#344]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Nextcloud php system information remains at 512M upload limit hardcoded in `00_opcache.ini` and I can't upload a 2GB file in a link-shared folder.

### Expected Behavior

`Nextcloud > Administration Settings > System > PHP > Upload max size` should change to 16G ish

### Steps To Reproduce

1. Add to the mounted `config/php/php-local.ini` file:
```
upload_max_filesize 16G
post_max_size 16G
max_input_time 3600
max_execution_time 3600
```

2. Restart container
3. Create folder
4. Share folder with link, enabling write access
5. Upload large file

### Environment

```markdown
- OS: Archlinux
- How docker service was installed: pacman
```

### CPU architecture

x86-64

### Docker creation

```bash
version: "3.8"
services:
  nextcloud:
    image: lscr.io/linuxserver/nextcloud:latest
    container_name: nextcloud
    environment:
      - PUID=1000
      - PGID=1000
    volumes:
      - ./config:/config
      - ./data:/data
    ports:
      - 443:443
    restart: unless-stopped
```
```


### Container logs

```bash
None
```

**Comment by thechubbypanda** [comment]
I can't seem to find anywhere that the `/config/php` directory is referenced in the php configuration, could this be why?

**Comment by nemchik** [comment]
The PHP upload setting should be taking effect, but there's also nginx settings in default.conf

**Comment by thechubbypanda** [comment]
> The PHP upload setting should be taking effect, but there's also nginx settings in default.conf

Yeah unfortunately the PHP settings are not being considered... I've trawled through everything I can think of.  
With regards to nginx, I think I've gotten everything but regardless, the PHP limit is the main thing right now.

**Comment by nemchik** [comment]
Sorry for not getting back to you sooner. I just did a handful of testing and I cannot confirm this issue. 

---

Before:
```ini
; Edit this file to override php.ini directives and restart the container

date.timezone = America/Chicago
```
![image](https://github.com/linuxserver/docker-nextcloud/assets/725456/84eababf-83d1-4e4e-ab1d-9232801a5744)

---

After:
```ini
; Edit this file to override php.ini directives and restart the container

date.timezone = America/Chicago

upload_max_filesize = 16G
post_max_size = 16G
max_input_time = 3600
max_execution_time = 3600
```
![image](https://github.com/linuxserver/docker-nextcloud/assets/725456/597f4bbe-dc01-45f0-b9d3-a63ee308ca6a)

I did not edit my nginx config in that test, but what you need to look for is https://github.com/linuxserver/docker-nextcloud/blob/d2fd6fc539bc7cb7f49f0b384cf95274d20e0cc3/root/defaults/nginx/site-confs/default.conf.sample#L28-L30

**Comment by drizuid** [comment]
I followed the nextcloud documentation and also cannot reproduce this issue. OP hasn't replied to the last comment, so I will close this as non-reproducible.

**Comment by fouteox** [comment]
Hello, I have exactly the same problem. I'm behind the swag proxy, correctly configured with the original nextcloud.subdomain.conf file.

Mon proxy est lui même derrière le proxy cloudflare.

I also use AdGuard Home for my DNS.

Do you want me to open a new issue ?

**Comment by arktisk-varg** [comment]
Issue is still there. Latest docker image 33.0.3.2.

Set php-local.ini to:

```php
; Edit this file to override php.ini directives

date.timezone = Europe/Berlin 
max_execution_time = 600
``` 

Restarted container
But still no effect:

<img width="652" height="247" alt="Image" src="https://github.com/user-attachments/assets/ca58aaa8-a545-4b8d-9325-7cedf85a92c3" />


This issue should be reopened and investigated especially due to the relevance with the current max execution time errors in current versions
https://github.com/linuxserver/docker-nextcloud/issues/536
https://github.com/nextcloud/server/issues/57894

**Comment by nemchik** [comment]
> date.timezone` = Europe/Berlin 
> max_execution_time = 600

I'm wondering if you were trying to format markdown when posting this, or if you actually have that backtick in your file. If you do, that's probably breaking things.

You would want this:

```ini
date.timezone = Europe/Berlin 
max_execution_time = 600
```

NOT THIS:

```ini
date.timezone` = Europe/Berlin 
max_execution_time = 600
```
(notice, even the markdown syntax highlighting if you're viewing this on desktop does not properly highlight the bad example because of the backtick)

**Comment by arktisk-varg** [comment]
No, it was a typo when posting, i corrected the comment

**Comment by nemchik** [comment]
I haven't been running NC in a while, but I just did a fresh install and tested to confirm, it doesn't seem like changes made to `/config/php/php-local.ini` are being reflected in NC's Settings/Admin/System page. I know that did work at some point in the past (tested and confirmed and posted above), but I am not entirely sure why it is not working now. It actually seems like `/etc/php84/conf.d/nextcloud.ini` is taking priority over `/config/php/php-local.ini` for some reason, and I am not sure why.

This will need further investigation.
